common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -1154,3 +1156,29 @@ def test_user_not_enrolled_error(self):
             u"{user} is not enrolled in this course. Please check your spelling and retry.".format(user=new_user),
             self.certificates_section.certificate_invalidation_message.text
         )
+
+
+@attr('a11y')
+class LMSInstructorDashboardA11yTest(BaseInstructorDashboardTest):
+    """
+    LMS Instructor Dashboard Accessibility Test Class
+    """
+    def setUp(self):
+        browser = os.environ.get('SELENIUM_BROWSER', 'firefox')
+
+        with patch.dict(os.environ, {'SELENIUM_BROWSER': browser}):
+            super(LMSInstructorDashboardA11yTest, self).setUp()
+
+        self.course_fixture = CourseFixture(**self.course_info).install()
+        self.log_in_as_instructor()
+        self.instructor_dashboard_page = self.visit_instructor_dashboard()
+
+    def test_instructor_dashboard_a11y(self):
+        # a11y test added on 24 May 2016, but mostly disabled as there's a slew
+        # of issues on the Instructor Dashboard. We should aim to resolve these
+        # before 2 October 2016, and include the below exclusions.
+        self.instructor_dashboard_page.a11y_audit.config.set_scope(
```

Comment:
Isn't the intention of this pull request to solve for that particular case? I don't understand why we'd put the time into fixing it, without putting in tests to assure it stays that way (that is, to assure that there are no regressions when folks are making other changes to this page, to the styling, etc). I do think it's a good idea to consider this for the custom aXe ruleset we use, but again we have unit tests that assert markup on this page. Can't we use that in this case?